Van/veb/bash/swap/swap.sh

```diff
@@ -0,0 +1,12 @@
+echo "file1"
+cat file1
+echo "file2"
+cat file2
+echo "*****Swaping*****"
+x=$(cat file1)
+cp file2 file1
+echo "$x" > file2
+echo "file1"
+cat file1
+echo "file2"
+cat file2
```

Comment:
Only contents of the files should be swapped
Will be merged with this  incomplete version